sdks/python/build.gradle

```diff
@@ -204,19 +204,30 @@ task directRunnerIT(dependsOn: 'installGcpTest') {
 
 // Before running this, you need to:
 //
-// 1. build the SDK container:
+// 1. Build the SDK container:
 //
 //    ./gradlew -p sdks/python/container docker
 //
-// 2. start a local JobService, for example, the Portable Flink runner
+// 2. Either a) or b)
+//  a) If you want the Job Server to run in a Docker container:
+//
+//    ./gradlew :beam-runners-flink_2.11-job-server-container docker
```

Comment:
Still does not work: `Task 'beam-runners-flink_2.11-job-server-container' not found in root project 'beam'.`

Reply:
while this does: `./gradlew -p runners/flink/job-server-container docker`

Reply:
Should be `./gradlew :beam-runners-flink_2.11-job-server-container:docker`